pkg/cache/placeholder.go

```diff
@@ -50,6 +54,10 @@ func newPlaceholder(placeholderName string, app *Application, taskGroup v1alpha1
 			},
 		},
 		Spec: v1.PodSpec{
+			SecurityContext: &v1.PodSecurityContext{
+				RunAsUser:  &runAsUser,
+				RunAsGroup: &runAsGroup,
```

Comment:
What is the setting for the firs pod? Wouldn't be better to use the same setting instead of defining some default user and group?